public/js/routing/router.js

```diff
@@ -72,12 +78,23 @@ export default class Router {
       this._updateContent();
       return;
     }
-    const location = this._window.document.location.href;
+    const location = this._document.location.href;
     const route = this.findRoute(location);
     this._takeOverAnchorLinks(this._container);
     route.onAttached();
   }
 
+  _dispatchNavigateEvent(url, route) {
+    const event = this._document.createEvent('CustomEvent');
```

Comment:
Will keep this for now, while we decide if we use an external polyfill, create our own or keep using the old syntax.